vhdbuilder/packer/install-dependencies.sh

```diff
@@ -246,12 +246,11 @@ done
 #Azure CNI has binaries and container images for ARM64 from 1.4.13
 AMD64_ONLY_CNI_VERSIONS="
 1.2.7
+1.4.19
```

Comment:
1.4.19 has both amd64 and arm64 binaries, no need to add it here. otherwise, the amd64 vhd build will pull it twice.

Reply:
understood, thanks for explaining